rfdiffusion/inference/run.py

```diff
@@ -38,20 +20,20 @@ def make_deterministic(seed=0):
     random.seed(seed)
 
 
-@hydra.main(version_base=None, config_path="../config/inference", config_name="base")
-def main(conf: HydraConfig) -> None:
+def run_inference(conf: HydraConfig) -> None:
     log = logging.getLogger(__name__)
     if conf.inference.deterministic:
         make_deterministic()
 
     # Check for available GPU and print result of check
     if torch.cuda.is_available():
+        log.info("GPU is available")
```

Comment:
This seems a bit excessive if two lines later there's a more detailed message, or did it help you in debugging something?

Reply:
`is_available` can be very fast. Actually finding the GPU name requires initializing cuda, which is slow (~12s as I'm running it). I added this when I was debugging why parallelizing the tests across 6 GPUs only gave me like a 2x speedup. I can remove it if you still think it's excessive.